app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1352,6 +1352,172 @@ def test_invalid_experiment_with_branch_missing_feature_value(self):
             {"reference_branch": {"feature_value": ["This field may not be blank."]}},
         )
 
+    def test_bucket_namespace_warning_for_dupe_rollouts(self):
+        lifecycle = NimbusExperimentFactory.Lifecycles.CREATED
+        desktop = NimbusExperiment.Application.DESKTOP
+        channel = NimbusExperiment.Channel.NIGHTLY
+        targeting_config_slug = NimbusExperiment.TargetingConfig.MAC_ONLY
+        experiment1 = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle,
+            application=desktop,
+            channel=channel,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_108,
+            feature_configs=[NimbusFeatureConfigFactory(application=desktop)],
+            is_sticky=False,
+            is_rollout=True,
+            targeting_config_slug=targeting_config_slug,
+        )
+        experiment2 = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle,
+            application=desktop,
+            channel=channel,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_108,
+            feature_configs=[NimbusFeatureConfigFactory(application=desktop)],
+            is_sticky=False,
+            is_rollout=True,
+            targeting_config_slug=targeting_config_slug,
+        )
+
+        for branch in experiment1.treatment_branches:
+            branch.delete()
+        for branch in experiment2.treatment_branches:
+            branch.delete()
```

Comment:
[chain](https://docs.python.org/3/library/itertools.html#itertools.chain) is a handy little trick

```py
for branch in chain(experiment1.treatment_branches, experiment2.treatment_branches):
    branch.delete()
```

Reply:
🔥